src/nn/tfjs/TfjsModel.js

```diff
@@ -33,18 +31,9 @@ export default class TfjsModel {
     switch(this._model._backend) {
       case('WASM'): {
         if(tf.getBackend() != 'wasm'){
-          function _fixWasmPath(wasmPath) {
-            // Assume the wasm file is located in the same folder as webml-polyfill.js
-            for (let s of document.getElementsByTagName('script')) {
-              if (s.src.indexOf('webml-polyfill.js') !== -1) {
-                let parts = s.src.split('/');
-                parts[parts.length - 1] = wasmPath;
-                return parts.join('/');
-              }
-            }
-            return '';
-          }
-          setWasmPath(null, {'tfjs-backend-wasm.wasm': _fixWasmPath(wasmPath)});
+          // Assume the wasm file is located in the same folder as webml-polyfill.js
+          const wasmPathPrefix = window.location.protocol + "//" + window.location.host + "/dist/";
+          setWasmPaths(wasmPathPrefix);
```

Comment:
```js
setWasmPaths(wasmPathPrefix); 
```
equals to 
```js
setWasmPaths({
  'tfjs-backend-wasm.wasm': wasmPathPrefix + 'tfjs-backend-wasm.wasm",
  'tfjs-backend-wasm-simd.wasm':  wasmPathPrefix + 'tfjs-backend-wasm-simd.wasm",
  'tfjs-backend-wasm-threaded-simd.wasm': wasmPathPrefix + 'tfjs-backend-wasm-simd.wasm",
});
```
For simple, I invoked setWasmPaths function with prefix string parameter.